src/Core/Renderer/RendererDefinitions.cs

```diff
@@ -5,11 +5,11 @@
 // </copyright>
 //-----------------------------------------------------------------------------
 
-namespace WheelMUD.Core
-{
-    using System.Collections.Generic;
-    using WheelMUD.Interfaces;
+using System.Collections.Generic;
+using WheelMUD.Server;
 
+namespace WheelMUD.Core.Renderer
```

Comment:
I'm not really sold on these or Behaviors becoming their own namespace again.  I suspect it'll feel better if Interfaces also are not isolated from their more general namespace.  We used to have way too many interfaces before, and it was harder to discover things through IntelliSense and the like, so we actively removed a bunch of them.  As evidence of this, the current code's commented-out CreateCurrency command shows "using WheelMUD.Universe.Behaviors;" which you're resurrecting with this PR.

This is not a blocking concern for me, but something we can discuss.  :)